packages/common/utils/directories.ts

```diff
@@ -43,20 +50,28 @@ export function recursiveFilesSearch({
 
 export function getStaticPath() {
     let staticPath =
-        config.staticFolderPath || path.join(getProgramPath(), 'static');
+        config.staticFolderPath || path.join(getDataPath(), 'static');
 
     // replace ./static with normal path to the static with program path
     if (
         staticPath.toLowerCase() === './static' ||
         staticPath.toLowerCase() === '.\\static'
     )
-        staticPath = path.join(getProgramPath(), 'static');
-
+        staticPath = path.join(getDataPath(), 'static');
+    ensureDirectoryExists(staticPath);
     return path.resolve(staticPath);
 }
 
 export function getCachePath() {
-    return path.join(getProgramPath(), '.cache');
+    if (process.platform === 'linux') {
+        const xdgCacheHome =
+            process.env.XDG_CACHE_HOME || path.join(homedir(), '.cache');
+        const cachePath = path.join(xdgCacheHome, 'tosu');
+        ensureDirectoryExists(cachePath);
+        return cachePath;
+    } else {
```

Comment:
why you need else statement if you close under if guard condition at 71?

Reply:
In the first round I was using switch case with entire possible value listed. This one is left over after refactor to if. I will remove the else.